store/users_postgres.go

```diff
@@ -0,0 +1,66 @@
+package store
+
+import (
+	"database/sql"
+
+	"github.com/gitpods/gitpods"
+)
+
+type UsersPostgres struct {
+	db *sql.DB
+}
+
+func NewUsersPostgres(db *sql.DB) *UsersPostgres {
+	return &UsersPostgres{db: db}
+}
+
+func (s UsersPostgres) List() ([]*gitpods.User, error) {
+	rows, err := s.db.Query(`SELECT * FROM users;`)
```

Comment:
Same here, if you look at togo and drone you can build it more abstract and reusable